README.md

```diff
@@ -1,28 +1,29 @@
 # Ansible Modules for Dell Technologies Unity
 
 The Ansible Modules for Dell Technologies (Dell) Unity allow Data Center and IT administrators to use RedHat Ansible to automate and orchestrate the configuration and management of Dell Unity arrays.
+> **NOTE:** The Ansible Modules for Dell Unity are released on a monthly cadence starting July '23
```

Comment:
We're not sure that We'll be releasing a unity collection every month. Is it good to mention this statement?

Reply:
done